alerts/client.go

```diff
@@ -16,22 +16,20 @@ import (
 
 // Client for managing alerts configs.
 type Client struct {
-	client        PlatformClient
-	logger        platformlog.Logger
-	tokenProvider auth.ServerSessionTokenProvider
```

Comment:
Darin upstreamed some changes that remove the necessity of a separate token provider.